script/generate-json-schemas

```diff
@@ -4,5 +4,5 @@ set -eu
 
 cd "$(dirname "$0")/.."
 
-$(npm bin)/ts-json-schema-generator --path ./src/inputs.ts --type RepoArray --additional-properties true --out src/json-schemas/RepoArray.json
-$(npm bin)/ts-json-schema-generator --path ./src/inputs.ts --type Instructions --additional-properties true --out src/json-schemas/Instructions.json
+$(npm bin)/ts-json-schema-generator --path ./src/inputs.ts --type RepoArray --out src/json-schemas/RepoArray.json
```

Comment:
Good point. I hadn't considered how it would make it hard for us to change the types and was only thinking about making it match as closely as possible right now. I agree it would be more convenient to be able to pass the new inputs and then update the types here afterwards.